Mage.Sets/src/mage/cards/k/KioraTheTidesFury.java

```diff
@@ -0,0 +1,63 @@
+package mage.cards.k;
+
+import mage.abilities.Ability;
+import mage.abilities.LoyaltyAbility;
+import mage.abilities.costs.common.SacrificeTargetCost;
+import mage.abilities.effects.common.*;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.Duration;
+import mage.constants.SubType;
+import mage.constants.SuperType;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.game.permanent.token.KrakenToken;
+import mage.target.TargetPermanent;
+
+import java.util.UUID;
+
+/**
+ * @author Svyatoslav28
+ */
+public final class KioraTheTidesFury extends CardImpl {
+    private static final FilterCreaturePermanent filter = new FilterCreaturePermanent(SubType.KRAKEN, "Kraken");
```

Comment:
technically this should just be a FilterPermanent, not a FilterCreaturePermanent, because the card says "a Kraken" not "a Kraken creature"